_pages/about-us/teams/products-and-platforms.md

```diff
@@ -1,10 +1,12 @@
 ---
 title: Products and Platforms
 ---
-Products and Platforms is one of the 5 business units within 18F. We strive to incubate tools for digital government.
```

Comment:
We provide digital products and platforms to help our federal partners more effectively deliver on their missions?